src/reviewers.rs

```diff
@@ -204,17 +206,21 @@ impl Reviewers {
 
     pub fn to_author(&self, reviewer: &str) -> Result<Option<Author>, UnknownReviewer> {
         let skip = &[
```

Comment:
Alphabetically sorted and added various forms of "rustdoc" which Guillaume loved to use for finished FCPs ^^'